packages/edit-site/src/components/dataviews/add-filter.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	privateApis as componentsPrivateApis,
+	Button,
+	Icon,
+} from '@wordpress/components';
+import { chevronRightSmall, plus } from '@wordpress/icons';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { unlock } from '../../lock-unlock';
+import { OPERATOR_IN } from './in-filter';
+
+const {
+	DropdownMenuV2,
+	DropdownSubMenuV2,
+	DropdownSubMenuTriggerV2,
+	DropdownMenuItemV2,
+} = unlock( componentsPrivateApis );
+
+const VALID_OPERATORS = [ OPERATOR_IN ];
+
+export default function AddFilter( { fields, view, onChangeView } ) {
+	const filters = [];
+	fields.forEach( ( field ) => {
+		if ( ! field.filters ) {
+			return;
+		}
+
+		field.filters.forEach( ( filter ) => {
+			if ( VALID_OPERATORS.some( ( operator ) => operator === filter ) ) {
+				filters.push( {
+					field: field.id,
+					name: field.header,
+					operator: filter,
+					elements: [
+						{
+							value: '',
+							label: __( 'All' ),
+						},
+						...( field.elements || [] ),
+					],
+					isVisible: view.filters.some(
+						( f ) => f.field === field.id && f.operator === filter
+					),
+				} );
+			}
+		} );
+	} );
+
+	if ( filters.length === 0 ) {
+		return null;
+	}
+
+	return (
+		<DropdownMenuV2
+			label={ __( 'Add filter' ) }
+			trigger={
+				<Button
+					aria-disabled={ filters.length === view.filters?.length }
```

Comment:
Should this be `__experimentalIsFocusable disabled` like suggested by @ciampo

Reply:
https://github.com/WordPress/gutenberg/pull/55992/commits/c10be614418ab9d36c169d684cd12df5e325f0ef

Reply:
Unfortunately I don't have much context around it, since these changes were discussed before I started working on `@wordpress/components`. Best I could find is [this comment](https://github.com/WordPress/gutenberg/pull/19337#issuecomment-572788121) — it looks like we initially wanted to make the prop `true` by default, but we were concerned about potential unintended consequences.

Reply:
Good call and I was on that discussion as well. I guess now, I may be more on the side of let's try and remove that prop and make it default and see how it goes.